scaletest/terraform/action/gcp_clusters.tf

```diff
@@ -139,6 +139,11 @@ resource "google_container_node_pool" "node_pool" {
     metadata = {
       disable-legacy-endpoints = "true"
     }
+    kubelet_config {
+      cpu_manager_policy = ""
+      cpu_cfs_quota      = false
+      pod_pids_limit     = 0
+    }
```

Comment:
Interesting, what made you need to set this?

Reply:
I was getting drift on created clusters with this set